ome_zarr/reader.py

```diff
@@ -278,14 +280,15 @@ def __init__(self, node: Node) -> None:
         axes_values = {"t", "c", "z", "y", "x"}
         try:
             multiscales = self.lookup("multiscales", [])
-            version = multiscales[0].get(
+            self.metadata["version"] = multiscales[0].get(
                 "version", "0.1"
             )  # should this be matched with Format.version?
             datasets = multiscales[0]["datasets"]
             # axes field was introduced in 0.3, before all data was 5d
             axes = tuple(multiscales[0].get("axes", ["t", "c", "z", "y", "x"]))
             if len(set(axes) - axes_values) > 0:
                 raise RuntimeError(f"Invalid axes names: {set(axes) - axes_values}")
+            self.metadata["axes"] = axes
```

Comment:
Adding the same metadata to `self` and `node` feels a bit odd, like it may make other uses downstream less clear. It might point to the internals needing to be simplified/refactored.

Reply:
Yes, you're correct. Re-reading this I am unsure of what is supposed to be written there. I'll close and we can get back to this when we need it.